src/cli/index.ts

```diff
@@ -181,6 +181,9 @@ export default class Cli {
       supportCodeLibrary,
     })
     await emitMetaMessage(eventBroadcaster)
+    if (configuration.featurePaths.length === 0) {
+      return { shouldExitImmediately: true, success: true }
+    }
```

Comment:
I disagree with this change. I think we run though the rest of things normally otherwise the formatters won't write that 0 scenarios / steps ran (and the user just gets no output)

Reply:
Agreed.

I added the above snippet because removing caused the following timeout error:

```
 When I run cucumber-js with `@test.txt` # features/step_definitions/cli_steps.ts:19
       Error: function timed out, ensure the promise resolves within 10000 milliseconds
           at Timeout._onTimeout (/home/lemur/code/cucumber-js/src/user_code_runner.ts:80:18)
           at listOnTimeout (internal/timers.js:554:17)
           at processTimers (internal/timers.js:497:7)
```
If I set `pickleIds` to [] when `configuration.featurePaths.length === 0` things seem to work as expected without the unexpected side effect of no formatter output.

Are there other side effects to setting `pickleIds` to `[]` when there are no features to run, that I am not seeing?